index.html

```diff
@@ -1,85 +1,65 @@
-                <li> <a href="#">ABOUT ME </a></li>
-                <li> <a href="#">RESUME</a> </li>
-                <li> <a href="#">PROJECTS</a></li>
-                <li> <a href="#">CONTACT</a></li>
-            </ul>     
-        </nav>
-
-        <div class= "content">
-
-            <div class="box box1 "> 
-
-                <img src="C:\Users\PCH.TL1\Desktop\Portfolio\pics\passport.jpeg" alt="passport" 
-                style="width:200px;height:200px;">
-                
-                <h3>Vincent Abuya Obunga </h3>
-                <h5>Software Developer </h5>
-                <h5>vabuyia@gmail.com</h5>
-            </div>
-
-            <div class="box box2 ">
-
-                <div class="about">
-                    
-                    <h4>ABOUT ME</h4>
-                    <p>I was born in 1983. I live in Basra and work as a Software Developer/Architect. <br> 
-                        I’ve been a professional developer for more than 2 years. <br> 
-                        I’ve mostly worked professionally with HTML, CSS, JavaScript, Ruby, Perl and PHP,<br> 
-                        but also little bit with Java and Python. C, C++. 
-                    </p>                         
-                </div>
-
-                <div class="social-icon">
-
-                    <h4> CONTACT </h4>
-
-                    <a href="https://twitter.com/abuyiag" target="_blank"> 
-                        <ion-icon name="logo-twitter"></ion-icon>  
-                    </a>
-
-                    <a href="linkedin.com/in/vincent-abuya-a1940555" target="_blank"> 
-                        <ion-icon name="logo-linkedin"></ion-icon> 
-                    </a>   
-
-                    <a href="https://github.com/vabuyia" target="_blank"> 
-                        <ion-icon name="logo-github"></ion-icon>
-                    </a> 
-                    <a href="https://github.com/vabuyia" target="_blank"> 
-                        <ion-icon name="logo-github"></ion-icon>
-                    </a>
-                                        
-                </div> 
-
-            </div>
-
-        </div>
-
-                          
-    </div>
-
-        
-    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
-    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
+ <header>
+   <a href="https://twitter.com/abuyiag"><img src="images/logo.png" alt=" " class="logo "></a>
+
+   <nav> 
+     <ul>
+       <li> <a href="https://github.com/vabuyia" target="_blank" rel="noopener noreferrer" >Portfolio</a></li>
+       <li> <a href=" linkedin.com/in/vincent-abuya-a1940555" target="_blank" rel="noopener noreferrer"> About </a></li>
+       <li> <a href="https://twitter.com/abuyiag" target="_blank" rel="noopener noreferrer"> Contact </a></li>
+      </ul>
+    </nav>
+
+</header>
+
+<section class="wrapper">
+
+  <section class="upper">
```

Comment:
- [ ] You need to fix the styling for the mobile version.

![Captura de pantalla_2021-09-18_17-35-19](https://user-images.githubusercontent.com/53111464/133910201-350f1462-0ad1-4445-abb6-a73edb9c6463.png)

-  kindly read this article in order to learn how to set up a chrome dev tool to [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/).
- How to use [css media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp).